src/chapters/functions/creating-functions.rst

```diff
@@ -0,0 +1,425 @@
+Creating Functions
```

Comment:
There is a LOT of content on this page.  Perhaps we should consider splitting it in two?  Maybe start the next page with ``return`` statements?